### PR TITLE
[one-cmds] Add profiling option

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds profiling option to one-cmds.

Related: #6080 
Draft: #6186 
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/one-cmds/one-build.template.cfg [hunk 1]
```diff
@@ -19,6 +19,7 @@ converter_version=v1
 [one-optimize]
 input_path=inception_v3.circle
 output_path=inception_v3.opt.circle
+profiling_data_gen=False
```
[the remark this answers is not in the document]
Comment (mhs4670go) [reply compiler/one-cmds/one-build.template.cfg:22]: @seanshpark sounds good to me. But, actually, `profiling_data_gen` is the option of `circle2circle`.

[the remark this answers is not in the document]
Comment (seanshpark) [reply compiler/one-cmds/one-build.template.cfg:22]: > profiling_data_gen is the option of circle2circle.

Ah......  maybe we can revise `circle2circle` if you like to use `generate_~~~` :)

**Review by seanshpark — DISMISSED** [review]
LGTM

File: compiler/one-cmds/how-to-use-one-commands.txt [hunk 2]
```diff
@@ -170,6 +170,7 @@ Current transformation options are
   Note that this pass can change the execution result of the model.
   So, use it only when the impact is known to be acceptable.
 - mute_warnings : This will turn off warning messages.
+- profiling_data_gen : This will turn on profiling data generation.
```
Comment (llFreetimell) [inline compiler/one-cmds/how-to-use-one-commands.txt:173]:
```suggestion
- generate_profile_data : This will turn on profiling data generation.
```

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**